ui/packages/file/src/FileUpload.svelte

```diff
@@ -44,7 +44,13 @@
 
 	let accept_file_types = "";
 	try {
-		file_types.forEach((type) => (accept_file_types += type + "/*, "));
+		for (const type of file_types) {
+			if (type.includes(".")) {
+				accept_file_types += type + ", ";
```

Comment:
there will be a trailing comma at the end if you're adding a comma after every type, I don't know how that is supported with every browser. Would be cleaner to create a list and use `list.join(", ")` to only insert commas in between.